custom_dc/custom_data.md

```diff
@@ -242,22 +242,24 @@ The `sources` section is optional. It encodes the sources and provenances associ
 
 To load custom data uploaded to Google Cloud, see instead [Pointing the local Data Commons site to the Cloud data](/custom_dc/data_cloud.html) for procedures.
 
+### Configure custom directories
+
+Edit the `env.list` file as follows:
+- Set the `OUTPUT_DIR` variable to the directory where your input files are stored. The load step will create a `datacommons` subdirectory under this directory.
```

Comment:
Consider adding a short note to explain this dichotomy. Something along the lines of a separate utility for managing data coming soon where input and output dirs can be different.

Reply:
Nah, let's not bother. Too much to explain now.